### PR TITLE
feature:add giant-squid navbar

### PR DESCRIPTION
**Pull request by osangel**
Add giant-squid navbar and calamari navbar will change when giant-squid acitivity start
UI:https://www.figma.com/file/IlNFZbcudqcF8qjugLPP85/Giant-Squid?node-id=187%3A2&t=qMcesxenGevMPU1e-0
image

### CODE REVIEW

File: src/styles/fonts.css [hunk 1]
```diff
@@ -11,3 +11,21 @@
...
+}
+@font-face {
+  font-family: 'RedHatText';
+  src: url('assets/fonts/RedHatText-Italic-VariableFont_wght.ttf');
+  font-style: italic;
+}
+@font-face {
+  font-family: 'RedHatText';
+  src: url('assets/fonts/RedHatText-VariableFont_wght.ttf');
+}
```
Comment (zhangjing28) [inline src/styles/fonts.css:31]: The font is supported by default. You don't have to add it on purpose

**Review by zhangjing28 — COMMENTED** [review]
The `Red Hat Text` and `Red Hat Mono` are supported by default, pls remove font files that related

File: src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    if (giantSquidTimeDuring()) {
+      setNavbarType(NAVBAR_TYPE.GIANT_SQUID);
+    } else {
+      setNavbarType(NAVBAR_TYPE.DEFAULT);
+    }
+  }, []);
+  return {
+    [NAVBAR_TYPE.GIANT_SQUID]: giantSquidNav,
+    [NAVBAR_TYPE.DEFAULT]: defaultNav,
+  }[navbarType];
```
[the remark this answers is not in the document]
Comment (osangel) [reply src/components/Navbar/index.tsx:41]: Each activity may need to add navbar, so I replaced it with an enumeration

[the remark this answers is not in the document]
Comment (zhangjing28) [reply src/components/Navbar/index.tsx:41]: Yeah, but I think `if` condition is better here

[the remark this answers is not in the document]
Comment (osangel) [reply src/components/Navbar/index.tsx:24]: All right,I will change it

File: src/styles/fonts.css [hunk 3]
```diff
@@ -1,13 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Text&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Mono&display=swap');
```
[the remark this answers is not in the document]
Comment (osangel) [reply src/styles/fonts.css:3]: Sorry，I forgot delete ttf documents。I don't think I should delete these css link because they are necessary

**Pull request merged**